tests/test_api.py

```diff
@@ -370,97 +374,94 @@ def test_node_dependencies(self, client, data_model_jsonld, source_node, return_
                 assert "TissueStatus" in response_dt
 
 @pytest.mark.schematic_api
-class TestManifestOperation:
-
+class TestManifestOperation():
     def ifExcelExists(self, response, file_name):
-        # return one excel file
+        # check if excel file gets returned as an attachment
         d = response.headers['content-disposition']
         fname = re.findall("filename=(.+)", d)[0]
         assert fname == file_name
     
-    def ifGoogleSheetExists(self, response_dt):
+    def ifGoogleSheetExists(self, response):
```

Comment:
Why camel case? The rest of the codebase is mostly snake case.

Reply:
This is a good point. Thank you Bruno. I will update the function to use snake case.